packit/api.py

```diff
@@ -129,7 +129,7 @@ def sync_release(
         assert_existence(self.up.local_project)
         assert_existence(self.dg.local_project)
 
-        self.package_config.run_action(action_name="pre-sync")
+        self.up.run_action(action_name="init")
```

Comment:
are you sure with the `"init"`?

Reply:
Any better name for that?

Reply:
No, I just thought it should be "pre-sync".

Reply:
I've needed to add two hooks:
- after cloning of the repo
- after checkout of the release branch

I am not sure:
- If both of them are needed. (But, why not have them.)
- How to name/distinguish them.

Reply:
how about `post-upstream-clone`? it's long but clear